industry_benchmarks/utils/plan_rbfe_network.py

```diff
@@ -256,7 +263,24 @@ def run_inputs(ligands, pdb, cofactors, output):
                 protocol=rbfe_protocol,
                 name=name
             )
-            transformation.dump(output / f"{transformation.name}.json")
+            transformations.append(transformation)
+
+    # Create the alchemical network and write it to disk
+    alchemical_network = openfe.AlchemicalNetwork(transformations)
+    alchemical_network_json_fp = output / "alchemical_network.json"
+    json.dump(
+        alchemical_network.to_dict(),
+        alchemical_network_json_fp.open(mode="w"),
+        cls=tokenization.JSON_HANDLER.encoder
+    )
+
+    # Write out each transformation
+    # Create a subdirectory for the transformations
+    transforms_dir = pathlib.Path(output / "transformations")
```

Comment:
Also here `"transformations"` 

looking at the bash scripts we should be fine, I am just double checking we don't make an assumption about the name of something if we let the user set it

Reply:
always transformations - that's based on the quickrun behaviour :)